### PR TITLE
Add the GoogleMaps API

### PR DESCRIPTION
**Pull request by Doges**
thanks @metehus :hahaa:

### CODE REVIEW

File: src/apis/GoogleMaps.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      let search = await this.request('/geocode', { address, language })
+      if (search.status === 'OK') {
+        let result = search.results[0]
+        if (!result.address_components.find(res => res.types.includes('administrative_area_level_2') || res.types.includes('locality'))) { resolve(null) }
+        resolve(result)
+      } else resolve(null)
+    })
+  }
+
+  getTimezone (location, language = 'en-us') {
```
Comment (pedrofracassi) [inline src/apis/GoogleMaps.js]: Document this function too

Comment (Doges) [reply src/apis/GoogleMaps.js]: whooooooooops, i sent it wrong

**Comment by metehus** [comment]
ok BUT i did the jsdoc stuff

**Comment by pedrofracassi** [comment]
@metehus approve it :haa

**Pull request merged**